pitop/processing/algorithms/line_detect.py

```diff
@@ -30,11 +32,11 @@ def calculate_blue_limits():
     return lower_blue, upper_blue
 
 
-def process_frame_for_line(frame, image_format="PIL", scale_factor=0.5):
+def process_frame_for_line(frame, image_format="PIL", process_image_width=320):
```

Comment:
Why?

Reply:
The scale factor is not useful since it will change performance depending on image passed in - best to scale to a known width so performance always gauranteed